tracing-honeycomb/src/reporter.rs

```diff
@@ -0,0 +1,45 @@
+use libhoney::FieldHolder;
+use std::collections::HashMap;
+
+#[cfg(feature = "use_parking_lot")]
+use parking_lot::Mutex;
+#[cfg(not(feature = "use_parking_lot"))]
+use std::sync::Mutex;
+
+/// Reports data to some backend
+pub trait Reporter {
```

Comment:
since the stuff in this module is re-exported, it should be "pub crate" ? :)

Reply:
Hmm, if we set this to `pub(crate) trait Reporter { /*... */}`  we cannot mark  our `Builder::build`-function as public, as it is generic over `R: Reporter`. I guess Reporter must be pub? Or did you mean something else?

Reply:
I thought the `pub use reporter::{LibhoneyReporter, Reporter, StdoutReporter};` in `lib.rs` would export it even if it is `pub(crate)` here, but I am at the edge of my knowledge here. So if you say it does not work like that I belive you. :)